.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature/Change Request
+about: A template for Druid feature and change requests
+title: "[REQUEST]"
```

Comment:
Nit, but, I'd remove the brackets from this one too.

Reply:
Removed